qucs/qucs/components/componentdialog.cpp

```diff
@@ -602,7 +602,8 @@ void ComponentDialog::slotSelectProperty(QTableWidgetItem *item)
     ButtAdd->setEnabled(true);
     ButtRem->setEnabled(true);
 
-    if (Comp->Description == "equation") {
+    // enable Up/Down buttons only for the Equation component
+    if (Comp->Model == "Eqn") {
```

Comment:
please explain how this "does not improve the situation". The purpose of this change is to fix a bug in the current code. If you think it doesn't not actually fix it, please propose a better/different way or point out why it does not fix it.